sound/soc/sof/topology.c

```diff
@@ -2408,6 +2408,8 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 	swidget->id = w->id;
 	swidget->pipeline_id = index;
 	swidget->private = NULL;
+	atomic_set(&swidget->use_count, 0);
```

Comment:
I don't see where use_count is modified by this patch. Something's missing or not explained, it's hard to follow if this reference count is well managed.

Reply:
it is not managed in this patch. The use_count management is all part of the next patch that introduces the dynamic pipeline feature. This patch is simply to introduce the new fields and helper functions